wrapper/src/main/java/software/amazon/jdbc/util/telemetry/OpenTelemetryFactory.java

```diff
@@ -57,11 +57,23 @@ public void postCopy(TelemetryContext telemetryContext, TelemetryTraceLevel trac
   }
 
   public TelemetryCounter createCounter(String name) {
+    if (name == null) {
+      throw new IllegalArgumentException("name");
+    }
+    if (name.length() > 63) {
+      name = name.substring(0, 63);
```

Comment:
do we have the full instance endpoint name in our efm counters? should we just use instance id instead?

Reply:
Instance Id isn't available in some case so complete instance endpoint is used instead.